templates/bridge-interface.j2

```diff
@@ -11,9 +11,14 @@ iface {{ openvpn_dev }} inet manual
 # Bridge
 auto br-{{ openvpn_dev }}
 iface br-{{ openvpn_dev }} inet static
-       bridge_ports {{ openvpn_dev }}
+       bridge_ports {% if 'ports' in openvpn_bridge %}{{ openvpn_bridge.ports }}
+       {% else %}{{ openvpn_dev }}
+       {% endif -%}
        bridge_stp off
        address {{openvpn_bridge.address}}
        netmask {{openvpn_bridge.netmask}}
        network {{openvpn_bridge.network}}
        broadcast {{openvpn_bridge.broadcast}}
+       {% for line in openvpn_bridge.script -%}
+       {{ line }}
+       {% endfor %}
```

Comment:
@conorsch you are right, I was not thinking about it. Yes, keeping backward compatibility is better, I would prefer it, what do you think @jangrewe?